e2e/cypress/integration/enterprise/elasticsearch_autocomplete/system_console_spec.js

```diff
@@ -52,20 +52,11 @@ describe('Elasticsearch system console', () => {
         // # Small wait to ensure new row is added
         cy.wait(TIMEOUTS.HALF_SEC);
 
-        // * First row should now say Pending
+        // # Get the first row
         cy.get('.job-table__table').
             find('tbody > tr').
             eq(0).
-            as('firstRow').
-            find('.status-icon-warning', {timeout: TIMEOUTS.HALF_MIN}).
-            should('be.visible').
-            and('have.text', 'Pending');
-
-        // * First row should update to say In Progress
-        cy.get('@firstRow').
-            find('.status-icon-warning', {timeout: TIMEOUTS.TWO_MIN}).
-            should('be.visible').
-            and('have.text', 'In Progress');
+            as('firstRow');
```

Comment:
I'm removing these checks on changing status since these are good to have only and not predictable. Will probably improve this in the future by stubbing related API request.